src/app/common/directives.js

```diff
@@ -1,40 +1,38 @@
-(function () {
+(function() {
```

Comment:
Style guides I've read before have suggested that. So that is how I've done them in my projects. Didn't know about the linter rules.